frontend/src/App.js

```diff
@@ -188,8 +188,7 @@ function App() {
   };
 
   const admin = isAdmin(user);
-  const { flags } = user || {};
-  const hasTrainingReportDashboard = flags && flags.includes('training_reports_dashboard');
+  const hasTrainingReportDashboard = canSeeBehindFeatureFlag(user, 'training_reports_dashboard');
```

Comment:
Can't get over the name of this function feel like it should be **hasFeatureFlag()** or something lol